src/main/java/frc/robot/RobotContainer.java

```diff
@@ -276,13 +305,24 @@ private void configureButtonBindings() {
           },
           swerveDrive));
 
+
+      fieldCentricButton.onTrue(new InstantCommand(
+          () -> {
+            swerveDrive.toggleFieldCentric();
+          }));
+
       fieldCentricButton.onTrue(new InstantCommand(
           () -> {
             System.out.println("FIELD CENTRIC TOGGLED");
             swerveDrive.toggleFieldCentric();
           }, swerveDrive));
       
       balanceChargingStationButton.whileTrue(levelChargingStationCommand);
+      balanceChargingStationButton.whileTrue(new InstantCommand(
+          () -> {
+            ledSubsystem.balancingColor();;
```

Comment:
Move this to the LevelChargingStationCommand, instead of attaching it to the button.